src/sentry/options/defaults.py

```diff
@@ -1584,10 +1584,16 @@
     flags=FLAG_MODIFIABLE_RATE | FLAG_AUTOMATOR_MODIFIABLE,
 )
 
-register("hybrid_cloud.outbox_rate", default=0.0, flags=FLAG_AUTOMATOR_MODIFIABLE)
```

Comment:
I didn't find any instances of this option in sentry, getsentry or options-automator.

Reply:
I vaguely remember this being used for backfills at one point? It's probably safe to remove now